monitor/README.md

```diff
@@ -0,0 +1,44 @@
+# Monitor plugin
+
+Monitors the health of your peers and helps you to decide if you might want to close a channel
+
+## Installation
+
+For general plugin installation instructions see the repos main
+[README.md](https://github.com/lightningd/plugins/blob/master/README.md#Installation)
+
+## Example Usage
+
+Unfortunately the python plugin framework doesn't pretty-print, nor does
+lightning-cli, so best viewed with -H (sorry for all the slasshes in the output. nothing I can do about it at this point):
```

Comment:
Wouldn't piping it into `jq` or `json_pp` be better?

Reply:
yes but not everyone has those installed / knows to use them. I wrote the plugin for a person who wanted to have as little explanation as possible I was afraid that installing jq would be too much of a hussle